sauceclient.py

```diff
@@ -57,9 +66,16 @@ def request(self, method, url, body=None):
                             (response.status, response.reason))
         return json_data
 
+    def get_encoded_auth_string(self):
+        auth_info = '%s:%s' % (self.sauce_username, self.sauce_access_key)
+        if is_py2:
+            base64string = base64.encodestring(auth_info)[:-1]
+        else:
+            base64string = base64.b64encode(auth_info.encode(encoding='UTF-8')).decode(encoding='UTF-8')
+        return base64string
 
-class Jobs(object):
 
+class Jobs(object):
```

Comment:
That's [your own code](https://github.com/cgoldberg/sauceclient/blame/master/sauceclient.py#L61) there; it's only a whitespace change. Also, it will ensure that it behaves the same way in Python 2 as in Python 3.